app/src/Pages/Portal/Contributions/Utils/ContributionsSections.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { shallow } from "enzyme";
+import {
+    OtherDetailsSection,
+    ContributorSection,
+    BasicsSection,
+    HeaderSection
```

Comment:
HeaderSection does not exist in ContributionsSections, looks like it was changed to AddHeaderSection or ReadyHeaderSection